src/scenes/Game.module.css

```diff
@@ -6,7 +6,6 @@
   display: flex;
```

Comment:
can this also be moved to `scenes/game/`?

Reply:
I'll change everything to sass.